src/App.jsx

```diff
@@ -1,7 +1,6 @@
 import Header from "./components/Header";
 import Footer from "./components/Footer";
 import Main from "./components/Main";
-import "./index.css";
 
 const App = () => {
   return (
```

Comment:
What do you need a class for that doesn't style anything?
You can use [react fragment](https://reactjs.org/docs/fragments.html#usage) or a [short version](https://reactjs.org/docs/fragments.html#short-syntax) for empty wrapping.